lib/pco_api/people/person.ex

```diff
@@ -20,4 +19,15 @@ defmodule PcoApi.People.Person do
   linked_association :phone_numbers
   linked_association :school
   linked_association :social_profiles
+
+  def list, do: get("people/v2/people")
+  def list(params) when is_list(params), do: get(params, "people/v2/people")
+
+  def get(id) when is_integer(id), do: get("people/v2/people/#{id}")
+
+  def create(%PcoApi.Record{attributes: _, type: _} = record), do: create(record, "people/v2/people")
```

Comment:
Feels kind of heavy to have to type so much to create a person.
`%PcoApi.Record{attributes: %{"first_name" => "geo", "last_name" => "lessel"}, type: "Person"} |> PcoApi.Person.create`

Can we just accept a map for creating a person providing the keys match the struct?
Then we could just do:
`PcoApi.Person.create(%{first_name: "geo", last_name: "lessel"})`